velox/exec/fuzzer/WindowFuzzer.h

```diff
@@ -80,9 +80,42 @@ class WindowFuzzer : public AggregationFuzzerBase {
  private:
   void addWindowFunctionSignatures(const WindowFunctionMap& signatureMap);
 
+  std::tuple<
+      core::WindowNode::WindowType,
+      core::WindowNode::BoundType,
+      core::WindowNode::BoundType>
+  generateFrameClause();
+
+  template <TypeKind TKind>
+  void addOffsetColumnsToInput(
+      std::vector<RowVectorPtr>& input,
+      core::WindowNode::BoundType startBoundType,
+      core::WindowNode::BoundType endBoundType,
+      SortingKeyAndOrder& orderByKey);
+
+  // Add offset column to input data for k-range frames. Returns the value of K
+  // as a string.
```

Comment:
In the comment,  let's also mention that this method generate a random constant k for the frame bound and update `input` with a column of the adjusted offset between k and the sorting key value.